sr_robot_launch/README.md

```diff
@@ -38,51 +93,40 @@ Trying to read mapping for: /hand/mapping/1178
 
 In this case 1178 is the serial number of the hand.
 
-### Real Robot hand only
-
-To start the hand without an arm:
-
-```bash
-roslaunch sr_robot_launch right_srhand_ur10arm.launch sim:=false arm_ctrl:=false arm_trajectory:=false hand_serial:=1178
-```
-
-or, for the left hand
-
-```bash
-roslaunch sr_robot_launch left_srhand_ur10arm.launch sim:=false arm_ctrl:=false arm_trajectory:=false hand_serial:=1178
-```
-
-### Real Robots, using the normal (not limited) joint range
+#### Real Robots, using the normal (not limited) joint range
 
 By default the URDF used for the UR10 arm uses a limited range for the joints, as that helps moveit find a planning solution. But as that restricts the robot movements, the user might want to start the robots with the full joint range. To do that:
 
 ```bash
-roslaunch sr_robot_launch right_srhand_ur10arm.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/right_srhand_ur10.urdf.xacro hand_serial:=1178
+roslaunch sr_robot_launch sr_right_ur10arm_hand.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/right_srhand_ur10.urdf.xacro hand_serial:=1178
 ```
 
 or, for the left hand and arm
 
 ```bash
-roslaunch sr_robot_launch left_srhand_ur10arm.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/left_srhand_ur10.urdf.xacro hand_serial:=1178
+roslaunch sr_robot_launch sr_left_ur10arm_hand.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/left_srhand_ur10.urdf.xacro hand_serial:=1178
 ```
 
-### Hand with Biotacs
+#### Hand with Biotacs
```

Comment:
We should have it for the standard PST (in case that's not done already).
And the plan is to have the OptoForce as standard in the near future.
Whether or not we have enough information (CAD-wise) to do it now, I'm not
sure. I'd recommend checking with Rob in case we're currently in a position
to do that work now.

On 9 December 2015 at 11:21, Beatriz Leon <notifications@github.com> wrote:

> In sr_robot_launch/README.md
> <https://github.com/shadow-robot/sr_interface/pull/116#discussion_r47078507>
> :
>
> >  ```
> >
> > -### Hand with Biotacs
> > +#### Hand with Biotacs
>
> We only have urdf xacros for biotacs, should I create more with other
> sensors? or just change the title and we will add more there when they are
> available?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shadow-robot/sr_interface/pull/116/files#r47078507>.
>



-- 
Armando De La Rosa T.
*Head of Dexterous Manipulation*
Tel: +44 (0) 20 7700 2487
armando@shadowrobot.com
www.shadowrobot.com

-- 

------------------------------
*Shadow Robot Company Ltd.*
251 Liverpool Road, N1 1LX, UK
Registered Number 3308007 (England & Wales)

RoNeX - Building Robots with ROS Made Easy 
http://www.shadowrobot.com/products/ronex/
------------------------------